src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -312,6 +313,20 @@ public void Initialize(IConsole console)
                         }
 
                         UpdateWorkingDirectory();
+
+                        if (!PowerShellLoaded)
+                        {
+                            var telemetryEvent = new TelemetryEvent("PowerShellLoaded", new Dictionary<string, object>
+                            {
+                                { "Trigger", console is IWpfConsole ? "PMC" : "PMUI" }
+                            });
+
+                            TelemetryActivity.EmitTelemetryEvent(telemetryEvent);
+                            PowerShellLoaded = true;
+                        }
+
+                        NuGetPowerShellUsage.RaisePowerShellLoadEvent(isPMC: console is IWpfConsole);
```

Comment:
you do the `console is IWpfConsole` check twice in this method. While this isn't a hot path where performance is critical, typecasting (the `is` operator) is slow, so consider doing it once, saving to a variable, and reusing the variable.